### PR TITLE
7 Essential Tools for .NET Developers 2017 Blog Post

### PR DESCRIPTION
**Pull request by leebrandt**
## Description:
- Describe your changes. This will most likely be a copy-paste of your commit
  message(s), which should be descriptive and informative.

## Type of Pull Request:

- [ ] Content (Documentation updates or typo-fixes)
- [x] Blog Post
- [ ] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:
* [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

## How Has This Been Tested?
- [x] I have built this locally and verified that it does not break existing functionality.
- [ ] For functional changes, I have tested against Chrome, Firefox, and Safari, and mobile.
- [ ] For functional changes, I have added tests.

## Primary Reviewers:

- *Tag reviewer(s)*

- [Blog Post Google Doc](https://docs.google.com)

### CODE REVIEW

**Comment by leebrandt** [comment]
Updated check boxes

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,1302 @@
+{
```
Comment (jmelberg-okta) [inline package-lock.json]: Is this file needed? Can we add it to our .gitignore?

File: blog/2017/06/27/seven-tools-net-developers.html [hunk 2]
```diff
@@ -0,0 +1,260 @@
+<!doctype html>
+<!--[if lt IE 7 ]> <html class="ie6 no-flexbox"> <![endif]-->
+<!--[if IE 7 ]>    <html class="ie7 no-flexbox"> <![endif]-->
+<!--[if IE 8 ]>    <html class="ie8 no-flexbox"> <![endif]-->
+<!--[if IE 9 ]>    <html class="ie9 no-flexbox"> <![endif]-->
+<!--[if IE 10 ]>    <html class="ie10 no-flexbox"> <![endif]-->
+<!--[if (gt IE 10)|!(IE)]><!--> <html class="modern wf-loading" lang="en"> <!--<![endif]-->
+  <head><head>
```
Comment (jmelberg-okta) [inline blog/2017/06/27/seven-tools-net-developers.html]: Looks like we have a weird Jekyll build issue casing two tags to be created here, but we only close one.

^ @rchild-okta

File: package-lock.json (hunk 1 above)
Comment (mraible) [inline package-lock.json]: It's a new file created by npm 5. It's similar to `yarn.lock` to lock down dependencies. The official docs recommend checking it in. https://docs.npmjs.com/files/package-lock.json

**Comment by jmelberg-okta** [comment]
🚀
